src/components/SelectAsync/index.tsx

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright (c) 2022 AccelByte Inc. All Rights Reserved.
+ * This is licensed software from AccelByte Inc, for limitations
+ * and restrictions contact your company contract manager.
+ */
+import React from "react";
+import { SelectProps as AkSelectProps, AsyncSelect, MenuListComponentProps } from "@atlaskit/select";
+import { components, FormatOptionLabelMeta, OptionTypeBase } from "react-select";
+import { SelectOption } from "../../types";
+import "../Select/index.scss";
```

Comment:
we used to find a problem where one css file is imported in multiple tsx files. it causes the css not injected. for now, please use [this](https://github.com/AccelByte/justice-ui-library/pull/75) solution as a workaround